packages/amazonq/package.json

```diff
@@ -554,6 +554,21 @@
             ]
         },
         "commands": [
+            {
+                "command": "aws.amazonq.stopCmdExecution",
+                "title": "Stop Amazon Q Command Execution",
+                "category": "%AWS.amazonq.title%"
```

Comment:
the stop shortcut will also used for global stop, right? if yes, then we should change the title. IMO

Reply:
not sure why no tool execution can be considered Q command stop / run / reject?